From 9ce4aab38747c80d043de5f907c7b514e2547df7 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Sat, 21 Mar 2015 11:18:05 -0400
Subject: [PATCH] Add back g_memmove

---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 67a766f..cdf800a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2928,10 +2928,8 @@ _______EOF
 
 
 	cat >>$outfile <<_______EOF
-#ifndef G_DISABLE_DEPRECATED
 #define g_ATEXIT(proc)	(atexit (proc))
 #define g_memmove(dest,src,len) G_STMT_START { memmove ((dest), (src), (len)); } G_STMT_END
-#endif
 $glib_defines
 $glib_os
 $glib_static_compilation
-- 
1.8.3.1

