diff -up boost_1_55_0/boost/spirit/home/support/char_set/range_run_impl.hpp\~ boost_1_55_0/boost/spirit/home/support/char_set/range_run_impl.hpp
--- boost_1_55_0/boost/spirit/home/support/char_set/range_run_impl.hpp~	2011-01-13 03:11:19.000000000 +0100
+++ boost_1_55_0/boost/spirit/home/support/char_set/range_run_impl.hpp	2014-05-15 18:09:07.326237692 +0200
@@ -25,9 +25,6 @@ namespace boost { namespace spirit { nam
         // if *iter intersects with, or is adjacent to, 'range'...
         if (can_merge(*iter, range))
         {
-            typedef typename Range::value_type value_type;
-            typedef integer_traits<value_type> integer_traits;
-
             // merge range and *iter
             merge(*iter, range);
 

Diff finished.  Thu May 15 18:09:11 2014
