diff -up ./boost/xpressive/detail/dynamic/parser.hpp~ ./boost/xpressive/detail/dynamic/parser.hpp
--- ./boost/xpressive/detail/dynamic/parser.hpp~	2009-09-23 23:53:53.000000000 +0200
+++ ./boost/xpressive/detail/dynamic/parser.hpp	2014-05-13 17:15:47.263001855 +0200
@@ -328,7 +328,6 @@ inline sequence<BidiIter> make_assert_en
 template<typename BidiIter, typename Cond, typename Traits>
 inline sequence<BidiIter> make_assert_word(Cond, Traits const &tr)
 {
-    typedef typename iterator_value<BidiIter>::type char_type;
     return detail::make_dynamic<BidiIter>
     (
         detail::assert_word_matcher<Cond, Traits>(tr)
