diff -up ./boost/archive/iterators/transform_width.hpp~ ./boost/archive/iterators/transform_width.hpp
--- ./boost/archive/iterators/transform_width.hpp~	2013-04-28 18:07:27.000000000 +0200
+++ ./boost/archive/iterators/transform_width.hpp	2014-05-13 18:08:53.683355277 +0200
@@ -119,8 +119,8 @@ public:
     transform_width(const transform_width & rhs) : 
         super_t(rhs.base_reference()),
         m_buffer_out_full(rhs.m_buffer_out_full),
-        m_remaining_bits(rhs.m_remaining_bits),
         m_buffer_in(rhs.m_buffer_in),
+        m_remaining_bits(rhs.m_remaining_bits),
         m_end_of_sequence(false)
     {}
 };
